android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -1009,6 +1013,19 @@ public void removeListeners(Integer count) {
 
     private  DailyAudioManager dailyAudioManager;
 
+    @ReactMethod
```

Comment:
i think device selection should maybe live in daily-specific files (with Daily in the name) and in daily-specific methods (again, with Daily in the name) like what we do with audio mode today. this'll make it easier down the road to keep track of what's ours v. what's from upstream.

Reply:
(this applies both to android + ios)

Reply:
For java, the WebRTCModule class is the entrypoint for all the methods that are invoked from React.

But I agree we can change the names from IOs and Android to include the Daily before on the new classes that we have created. Since we have agreed yesterday that we are not going to merge back to the main fork of the react-native-webrtc.

Reply:
Fixed